driver/src/main.rs

```diff
@@ -75,10 +74,11 @@ struct Options {
     ///     "alias": "USDC",
     ///     "decimals": 6,
     ///     "external_price": 1000000000000000000000000000000,
+    ///     "should_estimate_price": true
     ///   }
     /// }'
     #[structopt(long, env = "PRICE_FEED_INFORMATION", default_value = "{}")]
-    backup_token_data: TokenData,
+    backup_token_data: TokenFallbackData,
```

Comment:
Is this really "fallback" data? Isn't the information in there (except for the price) used as primary and not fallback data? Also given that the option name is called `PRICE_FEED_INFORMATION` I'd expect a name more similar (or equal) to that.

Reply:
Good point, I'll rename it.